tests/layer_tests/py_frontend_tests/test_torch_frontend.py

```diff
@@ -797,6 +797,61 @@ def forward(self, x):
     assert mm_num == 2
 
 
+def test_patched_8bit_model_converts():
+    from openvino.frontend.pytorch import patch_model
+    from openvino import convert_model, compile_model
+    from transformers.pytorch_utils import Conv1D
+
+    class ModelWithLinear(torch.nn.Module):
+        def __init__(self):
+            super().__init__()
+
+            self.branch1 = torch.nn.Sequential(
+                torch.nn.Embedding(10, 64),
+                torch.nn.Linear(64, 32),
+                torch.nn.ReLU()
+            )
+            self.branch2 = torch.nn.Sequential(
+                Conv1D(256, 128),
+                torch.nn.Linear(256, 64), torch.nn.ReLU()
+            )
+            self.buffer = torch.ones(32)
+
+        def forward(self, x1, x2):
+            out1 = self.branch1(x1)
+            out2 = self.branch2(x2)
+            return (out1 + self.buffer, out2)
+
+    example = (torch.randint(0, 10, [32, 64]), torch.randn(32, 128))
+
+    model_ref = ModelWithLinear().to(torch.float8_e4m3fn).float()
+    with torch.no_grad():
+        res_ref = model_ref(*example)
+    model_f8_e4m3 = model_ref.to(torch.float8_e4m3fn)
+    patch_model.__make_16bit_traceable(model_f8_e4m3)
```

Comment:
btw, why is it not called inside `convert_model`?

Reply:
This patching breaks the original model, we decided to call it only in controlled environment, like optimum